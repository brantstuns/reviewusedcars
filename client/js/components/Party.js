import React from 'react';

export default function Party() {
  return (
    <div className="party">
      <img src="http://cultofthepartyparrot.com/parrots/hd/middleparrot.gif" alt="🐧" />
    </div>
  );
}