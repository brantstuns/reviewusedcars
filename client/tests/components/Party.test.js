import React from 'react';
import { shallow } from 'enzyme';
import Party from '../../js/components/Party';

describe('Party Component', function () {
  let party;

  beforeEach(() => {
    party = shallow(<Party/>);
  });

  test('Party component should contain an img', () => {
    expect(party.find('img').length).toBe(1);
  });

  test('should match snapshot', () => {
    expect(party).toMatchSnapshot();
  });
});