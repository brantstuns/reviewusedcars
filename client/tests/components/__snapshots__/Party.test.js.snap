// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Party Component should match snapshot 1`] = `
<div
  className="party"
>
  <img
    alt="🐧"
    src="http://cultofthepartyparrot.com/parrots/hd/middleparrot.gif"
  />
</div>
`;
